### PR TITLE
Settings  to activate the logout mechanism.

### LINKED ISSUE #27: Adding logout button functionality

### PR DESCRIPTION
**Pull request by hearagmu**
In this contribution, we mainly try to code the logout process, making use of the API.
-We only navigate to the LogIn screen, if we obtain a code in the response JSON as confirmation.
 
-Also made some adjustments to add the constants in the master file.

 # Testing
Clone the repository and:
-Move to COVID10 folder
-run:
 npm install --force

#Recommendation
Use a cell phone for emulation, your experience testing the application will be better.
 
2021-05-03_20h20_59

### CODE REVIEW

File: COVID19/src/utils/constans.js [hunk 1]
```diff
@@ -19,13 +20,17 @@ const constans = {
   loginGreeting: 'Please, login and stay safe!',
   login: 'Login',
   forgotPW: 'Forgot password?',
+  errorLoginFail: 'Login failed!',
   urlAPI: 'https://hidden-cliffs-21927.herokuapp.com/api/user/sign_in',
   urlAPIregIncident: 'https://hidden-cliffs-21927.herokuapp.com/api/incident',
+  urlSingOut: 'https://hidden-cliffs-21927.herokuapp.com/api/user/sign_out',
```
Comment (fernandoamz) [inline COVID19/src/utils/constans.js]:
```suggestion
  urlAPI: 'https://hidden-cliffs-21927.herokuapp.com/api/user/sign_in',
  urlAPIregIncident: 'https://hidden-cliffs-21927.herokuapp.com/api/incident',
  urlSingOut: 'https://hidden-cliffs-21927.herokuapp.com/api/user/sign_out',
```

Only should be only one api route and concat the path that you want access for example 

```
urlApi: 'https://hidden-cliffs-21927.herokuapp.com/api/'
```

if you want access to path only concat the route for example 

```
`${urlApi}user/sign_in`
```

**Review by fernandoamz — CHANGES_REQUESTED** [review]

File: COVID19/src/utils/constans.js (hunk 1 above)
Comment (hearagmu) [reply COVID19/src/utils/constans.js]: You are absolutely right, I like the idea ... now i'm gonna solve it, I just have to do a replacement in the scripts that are involved.

File: COVID19/src/screens/main.js [hunk 2]
```diff
@@ -17,6 +19,38 @@ const Item = ({item, onPress}) => (
...
+          if (res.data.code === constans.signOutResDataCode)
+          {
+            navigation.navigate('Login');
+          }
+          else{
+            alert(constans.errorSignOut);
+          }  
+        }).catch((e) => {
+          alert(e.toString());
+        });
```
Comment (LeyaimMagma) [inline COVID19/src/screens/main.js]: can you create a services.js file and add the axios delete function?
example
```
export default {
  signIn: async (email, password) => {
    return axios.post(`URL`, {
      email,
      password,
    });
  },
};

```

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

File: COVID19/src/screens/main.js (hunk 2 above)
Comment (hearagmu) [reply COVID19/src/screens/main.js]: yes, good idea.

**Review by LeyaimMagma — APPROVED** [review]

**Review by fernandoamz — APPROVED** [review]

**Review by JorgeGarciaRa — APPROVED** [review]
Good job!

**Pull request merged**